contracts/andromeda_gumball/examples/schema.rs

```diff
@@ -0,0 +1,16 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use andromeda_protocol::cw20::{ExecuteMsg, InstantiateMsg, QueryMsg};
```

Comment:
This import should be for the types you use in the contract. So `andromeda_protocol::gumball::...`